### ISSUE TITLE
1.1 repo size

### ISSUE THREAD

**Issue by jaytay79** [berzerk0/Probable-Wordlists#13]
Following the comments about the reduced repo size I just tried to clone it but it still appears to be absolutely massive:

```
Cloning into 'Probable-Wordlists'...
remote: Counting objects: 1649, done.
remote: Compressing objects: 100% (7/7), done.
receiving objects:  31% (522/1649), 1.83 GiB | 6.67 MiB/s
```

I am assuming this is because the old versions of the files are still in the commit history, could they be removed using [BFG](https://rtyley.github.io/bfg-repo-cleaner/)?

**Comment by berzerk0** [comment]
That tool looks great - I'll do it promptly

**Comment by berzerk0** [comment]
You know, I think I found which files were causing the size difference.

![image](https://cloud.githubusercontent.com/assets/26533735/25688599/97c61d9a-304e-11e7-8032-1bae354afbee.png)

They got to go! Going to run BFG. Thanks, @jaytay79.

**Comment by berzerk0** [comment]
Current Zip Size: 54MB
Current Clone Size: 118 MB

Progress for sure, but I'm still trying to purge the clones of unwanted history. 
It may be that I can't push ALL the files away in the clones - so far I have encountered an error similar what is described [in this StackOverflow post](https://stackoverflow.com/questions/34265266/remote-rejected-errors-after-mirroring-a-git-repository) - where some answers may be found.

I'm continuing to chase the files down and have been directed to [this set of instructions](http://christoph.ruegg.name/blog/git-howto-mirror-a-github-repository-without-pull-refs.html) where they describe my error may come from pull refs.

In the past, there has only ever been one pull request from an outside source, and I've only ever created one branch to work on, which I ended up deleting.


**TL;DR - It's smaller now, but cloning is still 118 MB, less than half of which aren't hidden files.**
**Since the repo is mostly links, you can get by with the 54 MB zip download for now**
**The situation is not WHOLLY intolerable at this size, but I am continuing to try and shrink it**

**Comment by berzerk0** [comment]
Will run this again after torrents uploaded (within 5 days)

**Comment by berzerk0** [comment]
Pre Torrent upload:

Zip - 5.6 MB
Clone  - 74.7MB

Will run a BFG again after torrents go up
